docs/primer/index.rst

```diff
@@ -1,17 +1,57 @@
+   :figwidth: 85%
+   :align: center
+   
+Beneficial ownership can include all types of relationship with an entity such as ownership, control or benefitting from the entity’s activities. 
+
+If a person owns something, they usually have some control over it. Owning shares in a company may confer voting rights, for example. But ownership and control are sometimes separated - by share classes, contracts, agreements and other mechanisms. For example, an executive director might have the right to appoint 60% of the other board members, while having little financial stake in the business. There are also situations where a person may not have ownership or significant control but still has a relationship which means they benefit from the activities of an entity - for example, where they can enjoy and use assets owned by the entity.
+
+For more `information about beneficial ownership transparency <https://www.openownership.org/en/about/what-is-beneficial-ownership-transparency/>`__ and to read `the latest research <https://www.openownership.org/en/research/>`__, visit `the Open Ownership Website <https://www.openownership.org/en/>`__
+
+.. _whatisbods:
+
+What is the Beneficial Ownership Data Standard?
+-----------------------------------------------
+
+The Beneficial Ownership Data Standard (BODS) provides a structured data format, along with guidance for collecting, sharing and using data on beneficial ownership.
+
+
+Within BODS:
+
+* the :doc:`data schema<../schema/index>` describes how and what data should be shared. It can also inform the design of data collection and management systems. The schema is defined in a structured data format called JSON
+* the :doc:`technical guidance<../schema/guidance/index>` provides support for publishers and users of the data
+
+Collecting, sharing and using beneficial ownership information as `structured data <https://www.openownership.org/en/publications/structured-and-interoperable-beneficial-ownership-data/benefits-of-structured-and-interoperable-data/>`__ has a number of benefits. 
+
+Broadly, it:
+
+* improves the functionality of beneficial ownership information
+* reduces the cost of producing, using, and maintaining beneficial ownership information
+* has a greater chance of meeting beneficial ownership transparency policy goals than unstructured data
+
+Without common standards, combining and analysing data from different sources can be expensive and time-consuming, making it more difficult to understand beneficial ownership networks.
+
+BODS allows publishers to provide statements that describe:
+
+* :any:`details about natural persons <schema-person-record>`
```

Comment:
The links in this block of bullets will work with the following changes:

```

* :any:`details about natural persons <schema-person-record>`
* :any:`details about entities <schema-entity-record>`
* :any:`identifiers for natural persons and entities <guidance-identifiers>`
* :any:`types of interests that can be held in entities <schema-interest>`
* :any:`provenance information <schema-source>`
* :any:`historical, as well as current, information <representing-updates>`
* :any:`placeholders, when information must be witheld <schema-interested-party>`
```
BUT - I don't think it's necessarily helpful to readers to include these links. They lead to very specific bits of the standard which are unlikely to be that useful if someone is at the stage of reading the primer. I'd rather they went exploring via the other links in this section.

Reply:
Yes that's a fair point - it might be overwhelming to have this many links